File: app/src/main/java/ua/hospes/rtm/data/team/models/TeamDb.java
package ua.hospes.rtm.data.team.models;

import android.content.ContentValues;

import ua.hospes.dbhelper.BaseModelInterface;
import ua.hospes.rtm.core.db.tables.Teams;

/**
 * @author Andrew Khloponin
 */
public class TeamDb implements BaseModelInterface {
    private int id = -1;
    private String name;


    public TeamDb(int id, String name) {
        this.id = id;
        this.name = name;
    }


    //region Getters
    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }
    //endregion


    @Override
    public ContentValues toContentValues() {
        ContentValues cv = new ContentValues();

        cv.put(Teams.NAME, name);

        return cv;
    }
}

File: app/src/main/java/ua/hospes/rtm/data/team/TeamDb.kt
package ua.hospes.rtm.data.team

import android.content.ContentValues
import ua.hospes.dbhelper.IDbModel
import ua.hospes.rtm.core.db.tables.Teams

data class TeamDb(
        val id: Int? = null,
        val name: String
) : IDbModel {

    override fun toContentValues(): ContentValues = ContentValues().apply { put(Teams.NAME.name(), name) }
}